explore/src/main/scala/explore/targeteditor/SpectralDefinitionEditor.scala

```diff
@@ -197,7 +198,13 @@ sealed abstract class SpectralDefinitionEditorBuilder[
         props.catalogInfo.flatMap(ci =>
           ci.objectType.map(ot =>
             React.Fragment(
-              FormLabel(htmlFor = "catalogInfo".refined)(ci.catalog.shortName),
+              FormLabel(htmlFor = "catalogInfo".refined)(
```

Comment:
Would it be better to change the `Display` instance?

Reply:
It didn't seem right that the display instance for `CatalogName.Import` was "Object Type".

The thing is, we are collapsing two fields (catalog name and object info) into one. One is the label and the other the value. We could instead show 2 fields and then use the `Display` instance for the catalog name.

Reply:
👍